src/screens/ShotsScreen.tsx

```diff
@@ -0,0 +1,37 @@
+import React, { useEffect, useState } from "react";
+import { View, StyleSheet, Text, FlatList } from "react-native";
```

Comment:
`FlatList`, `useEffect` and `useState` don't seem to be used. They shouldn't be imported.